### PR TITLE
Add `get packages` sub-command

### PR DESCRIPTION
**Pull request by hodgiwabi**
* Unify `get` command output formatting
  * yaml-consistent three-dash start of file enables output yaml parsing
  * lowercase keys
  * key: value spacing for secrets.data and packages.spec.credentials

+ Add packages output template

* Common go:embed moved to root.go file

### CODE REVIEW

**Comment by cmoulliard** [comment]
@hodgiwabi Can you please create a github issue describing what this PR is aiming for and will fix, enhance, etc ?

**Comment by cmoulliard** [comment]
Don't forget to update the README.md page and enhance the test cases ;-)

File: pkg/cmd/get/templates/secrets.tmpl
Comment (nabuskey) [inline pkg/cmd/get/templates/secrets.tmpl:1]: Thanks for working on this. This template file is not meant to be YAML parseable though. I know it looks like one but it's meant for humans to read the outputs easily. We have a yaml output option for yaml parsing purposes: https://github.com/cnoe-io/idpbuilder/blob/e8e231942f0449cb7193bfcbc72f5e49358521b2/pkg/cmd/get/secrets.go#L179-L198

File: pkg/cmd/get/packages.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	Short: "retrieve package info from the cluster",
+	Long:  ``,
+	RunE:  getPackagesE,
+}
+
+type PackageTemplateData struct {
+	Name        string                  `json:"name"`
+	Namespace   string                  `json:"namespace"`
+	Category    string                  `json:"category"`
+	PackageType string                  `json:"packageType"`
```
Comment (nabuskey) [inline pkg/cmd/get/packages.go:22]: I honestly do not know if we can get this kind of data right now.

**Comment by nabuskey** [comment]
Closing this due to inactivity. Feel free to reopen.

**Pull request closed without merging**